tests/foreman/cli/test_repositories.py

```diff
@@ -124,3 +124,41 @@ def test_positive_disable_rh_repo_with_basearch(module_target_sat, module_entitl
         }
     )
     assert 'Repository disabled' in disabled_repo[0]['message']
+
+
+def test_positive_reclaim_space(module_target_sat, module_entitlement_manifest_org):
```

Comment:
I'd probably change the name so it is not too misleading - I believe you do not really test that any space has been reclaimed, since you only run it on an `on-demand`-synced repo, right?